crates/autopilot/src/run_loop.rs

```diff
@@ -270,6 +272,23 @@ impl RunLoop {
                 return;
             }
 
+            let surplus_capturing_jit_order_owners =
```

Comment:
Refeching all jit order owners here sounds wrong. What if a new AMM was deployed after the auction was dispatched to solvers? Then it should not be saved to disk.

Aren't those part of the `Auction` struct (they should IMO). As such we should just persist this field instead of fetching things again.

Reply:
@fleupold the PR is in a draft, @sunce86 and I are still working on it :pray:

Reply:
Sure, just wanted to point this out early to avoid spending time polishing something that may receive fundamental feedback.

Reply:
I see, it makes a lot of sense, thanks a lot for pointing it out! we were aware of it, and @sunce86 was preparing that part :pray: